app/src/main/java/woowacourse/movie/movie/activity/SeatSelectionActivity.kt

```diff
@@ -8,15 +11,23 @@ import androidx.appcompat.app.AlertDialog
 import androidx.appcompat.app.AppCompatActivity
 import domain.Seat
 import domain.Seats
-import woowacourse.movie.dto.MovieDateDto
-import woowacourse.movie.dto.MovieDto
-import woowacourse.movie.dto.MovieTimeDto
-import woowacourse.movie.dto.SeatsDto
-import woowacourse.movie.dto.TicketCountDto
-import woowacourse.movie.mapper.mapToSeats
-import woowacourse.movie.mapper.mapToSeatsDto
-import woowacourse.movie.view.SeatSelectView
+import woowacourse.movie.R
+import woowacourse.movie.movie.AlarmReceiver
+import woowacourse.movie.movie.AlarmReceiver.Companion.ALARM_CODE
+import woowacourse.movie.movie.AlarmReceiver.Companion.ALARM_TIME
+import woowacourse.movie.movie.AlarmReceiver.Companion.REQUEST_CODE
+import woowacourse.movie.movie.dto.BookingHistoryDto
+import woowacourse.movie.movie.dto.movie.BookingMovieDto
+import woowacourse.movie.movie.dto.movie.MovieDateDto
+import woowacourse.movie.movie.dto.movie.MovieDto
+import woowacourse.movie.movie.dto.movie.MovieTimeDto
+import woowacourse.movie.movie.dto.seat.SeatsDto
+import woowacourse.movie.movie.dto.ticket.TicketCountDto
+import woowacourse.movie.movie.mapper.seat.mapToSeats
+import woowacourse.movie.movie.mapper.seat.mapToSeatsDto
+import woowacourse.movie.movie.view.SeatSelectView
 import java.time.LocalDateTime
+import java.util.Calendar
```

Comment:
```suggestion
import java.util.Calendar
```
> java.util.Date 또는 java.util.Calendar와 같은 날짜/시간 처리 클래스는 JDK 1.0 이후로 꾸준히 사용되어 왔지만 최신 프로그램에서는 잘 사용하지 않는다.

이전 미션에서도 다뤘듯이, java.util.Calendar를 지양해보면 어떨까요?

ref. https://techcourse.woowahan.com/s/ZprkZTJf/ls/u8sn5Zaf

Reply:
공식 문서에 있는 예제로 확인되어 숨김처리하였습니다. 🙏